cmd/k8s-operator/deploy/manifests/operator.yaml

```diff
@@ -110,52 +128,51 @@ metadata:
   namespace: tailscale
 spec:
   replicas: 1
-  strategy:
-    type: Recreate
   selector:
     matchLabels:
       app: operator
+  strategy:
+    type: Recreate
   template:
     metadata:
       labels:
         app: operator
     spec:
+      containers:
+      - env:
+        - name: OPERATOR_HOSTNAME
+          value: tailscale-operator
+        - name: OPERATOR_SECRET
+          value: operator
+        - name: OPERATOR_LOGGING
+          value: info
+        - name: OPERATOR_NAMESPACE
+          valueFrom:
+            fieldRef:
+              fieldPath: metadata.namespace
+        - name: CLIENT_ID_FILE
+          value: /oauth/client_id
+        - name: CLIENT_SECRET_FILE
+          value: /oauth/client_secret
+        - name: PROXY_IMAGE
+          value: tailscale/tailscale:unstable
+        - name: PROXY_TAGS
+          value: tag:k8s
+        - name: APISERVER_PROXY
+          value: "false"
+        - name: PROXY_FIREWALL_MODE
+          value: auto
+        image: tailscale/k8s-operator:unstable
```

Comment:
Probably at some point we will want to hardcode this to 'latest' tag instead, but not in this PR.